api/files/views.py

```diff
@@ -36,10 +35,10 @@ def get_file(self, check_permissions=True):
 class FileDetail(generics.RetrieveUpdateAPIView, FileMixin):
     """Details about a specific file.
     """
+    __metaclass__ = OsfAPIViewMeta
+
     permission_classes = (
-        drf_permissions.IsAuthenticatedOrReadOnly,
         CheckedOutOrAdmin,
-        base_permissions.TokenHasScope,
```

Comment:
Not a big deal, but I think we can keep these in, even though OsfAPIViewMeta will ensure that they are there.

Devs shouldn't have to read metaclass code to understand the permissions applied to a class.